cluster/get-kube-local.sh

```diff
@@ -53,9 +54,32 @@ function run {
   fi
 }
 
+# Creates a kubeconfig file for the kubelet.
+# Args: address (e.g. "http://localhost:8080"), destination dir
```

Comment:
Why make the server address an argument? It's only ever set once (to localhost) and if you tried to set it to anything else, it would likely require client auth which isn't handled by this function. I'd rename this function to `create-local-kubelet-kubeconfig()` and only pass in the destination directory as an argument.